libs/contract/truffle-config.js

```diff
@@ -1,15 +1,17 @@
 'use strict';
-var path = require('path');
-const HDWalletProvider = require("@truffle/hdwallet-provider");
-// Allow using typescript in deployments
-loadTypescript();
 // Load environment for the network we are deploying to
 require('../../tools/setenv');
-const { AccountId } = require('@thecointech/signers');
-// Load directly from path to side-step the limitations imposed
-// by the new package.exports module resolution
-const { loadFromPK } = require('../signers/build/encrypted');
+require('../../__mocks__/mock_node');
+// Allow using typescript in deployments
+loadTypescript();
+
+var path = require('path');
+const HDWalletProvider = require("@truffle/hdwallet-provider");
+const { AccountId, getSigner } = require('@thecointech/signers');
 
+const numBuiltIn = AccountId.BrokerCAD + 1;
+const testAccounts = [];
+loadAccounts(numBuiltIn).then(v => testAccounts = v).catch(console.error);
```

Comment:
slightly cheat-y and quite risky.  No top-level async here, and we can no longer access signers directly (they are env-specific now).  So, just assume that we will load signers before the code finishes compiling

Reply:
side note: it's risky in that it might not work in production, but not risky in that if it does break (we'll have to see when this code gets to that point) it will be extremely obvious and nothing in the system will break